src/mainwindow.cpp

```diff
@@ -491,9 +491,14 @@ void MainWindow::on_actionSaveAs_triggered() {
 }
 
 void MainWindow::on_actionDelete_triggered() {
-  // delete the current file
-  if(currentFile_)
-    Fm::FileOperation::deleteFiles({currentFile_});
+  // delete or trash the current file
+  if(currentFile_) {
+    if(static_cast<Application*>(qApp)->settings().useTrash()) {
+      Fm::FileOperation::trashFiles({currentFile_}, false);
+    } else {
```

Comment:
Please also respect the original code style here:

```c++
    }
    else {
```

Reply:
done.